slice/Glacier2/Metrics.ice

```diff
@@ -14,7 +14,7 @@
 
 #include "Ice/Metrics.ice"
 
-["swift:module:Glacier2:MX"]
```

Comment:
We decided to just drop the `MX` prefix from these modules's contents.

Without the 2nd argument, `swift:module:Foo` and `swift:identifier:Foo` are equivalent.